script/upgrades/MINIDROP/deploy/MINIDROP-00-Create-MerkleDistributor.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.5.13;
+
+import { console } from "forge-std/console.sol";
+import { Script } from "script/utils/Script.sol";
+import { Chain as ChainLib } from "script/utils/Chain.sol";
+import { Contracts } from "script/utils/Contracts.sol";
+import { IRegistry } from "../../../interfaces/IRegistry.sol";
+import { Contracts } from "../../../utils/Contracts.sol";
+import { IGovernanceFactory } from "../../../interfaces/IGovernanceFactory.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+
+interface IOwnableLite {
+  function transferOwnership(address newOwner) external;
+}
+
+contract MINIDROP_CreateMerkleDistributor is Script {
+  using Contracts for Contracts.Cache;
+
+  function run() public {
+    contracts.load("MUGOV-00-Create-Factory", "latest");
+    IRegistry registry = IRegistry(0x000000000000000000000000000000000000ce10);
+    address cUSD = registry.getAddressForStringOrDie("StableToken");
+    address MENTO = IGovernanceFactory(contracts.deployed("GovernanceFactory")).mentoToken();
+
+    // Transfer ownership to mentoLabsMultiig for drain mechanism
+    address mentoLabsMultisig = contracts.dependency("MentoLabsMultisig");
+
+    // 91 days after the current block timestamp roughly 3 months
+    uint256 endTime = block.timestamp + 91 days;
+
+    bytes32 merkleRootCUSD = readMerkleRoot(".cUSDRoot");
+    bytes32 merkleRootMENTO = readMerkleRoot(".mentoRoot");
+
+    address cUSDDistributor;
+    address mentoDistributor;
+
+    vm.startBroadcast(vm.envUint("MENTO_DEPLOYER_PK"));
+    {
+      cUSDDistributor = deployMerkleDistributor(cUSD, merkleRootCUSD, endTime);
+      console.log("MerkleDistributor for cUSD deployed at:", cUSDDistributor);
+      mentoDistributor = deployMerkleDistributor(MENTO, merkleRootMENTO, endTime);
+      console.log("MerkleDistributor for MENTO deployed at:", mentoDistributor);
+
+      IOwnableLite(cUSDDistributor).transferOwnership(mentoLabsMultisig);
```

Comment:
No need to change but you could also do this in one line without creating the interface:

(bool success,) = cUSDDistributor.call(abi.encodeWithSignature("transferOwnership(address)",mentoLabsMultisig));